src/index.html

```diff
@@ -5,8 +5,372 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img src="images/logo.png" alt="moyoLogo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a class="is-active" href="#">apple</a>
+          </li>
+          <li class="nav_item">
+            <a class="nav_link" href="#">samsung</a>
+          </li>
+          <li class="nav_item">
```

Comment:
Add blank line between the links
```suggestion
          </li>
          
          <li class="nav_item">
```